### PR TITLE
Work section part pull

### PR DESCRIPTION
**Pull request by MishDM**
add all information on the work section of the portifolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+            <p class="listname">Skills</p>
+            <button type="button" class="icon">
+              <img src="images/Disabled-right.svg" alt="button3" />
+            </button>
+          </div>
+          <ul class="skillul"></ul>
+        </div>
+      </div>
+    </div>
+  </section>-->
```
Comment (sumairq) [inline index.html:258]: - [ ] Kindly remove this commented out code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+      <li>
+        <a href="#">
+          <img src="images/github.svg" alt="github" />
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="images/istock.svg" alt="istock" />
+        </a>
+      </li>
```
Comment (sumairq) [inline index.html:64]: - [ ] Good job 👍  , adhering to the Figma template for the most part, however you need to fix the social icons , they need a left margin for better alignment. And also you need to ensure that the lower left area of your design has that curved edge border like in the [Figma](ma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template.  
### Figma Design
image
### Your Design
image

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @MishDM , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] It would be nice to have a bit longer **description for the pull request**, your's is too short.📑
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @MishDM,

Good job so far! I know you have worked hard on implementing the project and its requirements. However, there is an issue that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
## Required Changes ♻️

- [ ] Please, ensure that all your linters have passed i.e every check should be green. Fix that and your project will be completed 💯😊 

- [ ] Kindly check the comments under the review.

## Optional suggestions

- [ ] It would be better if your buttons have the Figma button styling on `:hover`, `:active`, `:disbaled`, `:pressed`. Let's see the Figma template design for buttons:

image

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @MishDM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**